lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -862,6 +862,10 @@ abstract class DefaultTextEditingStrategy implements TextEditingStrategy {
       activeDomElement.setAttribute('type', 'password');
     }
 
+    if (config.inputType == EngineInputType.none) {
+      activeDomElement.setAttribute('inputmode', 'none');
+    }
```

Comment:
Not sure about this part. If `inputmode` was previously deliberately not set on desktop, should we not set `inputmode=none` either...?

Reply:
Yes, this is deliberate. Desktop browsers don't have a virtual keyboard, so `inputmode` is meaningless there.